src/content-blocks/BlockMediaList/BlockMediaList.tsx

```diff
@@ -29,61 +28,61 @@ export interface BlockMediaListProps extends DefaultProps {
 	elements: MediaListItem[];
 	navigate: (action: ButtonAction) => void;
 	orientation?: Orientation;
+	ctaTitle?: string;
+	ctaContent?: string;
+	ctaButtonAction?: ButtonAction;
```

Comment:
Bij `BlockCTA` ontwijk ik de nood aan een buttonAction omdat ik dit in de client al gebruik in de opbouw van de `navigate()` functie. Zo heb je minder logic in je component hier.

Reply:
Fixed